orbit/pkg/packaging/windows.go

```diff
@@ -149,19 +149,59 @@ func BuildMSI(opt Options) (string, error) {
 		}
 	}
 
-	if err := wix.Heat(tmpDir, opt.NativeTooling, opt.LocalWixDir); err != nil {
+	absWixDir := opt.LocalWixDir
+	wineChecked := false
+
+	// Download wix for macOS running on arm64, unless a local-wix-dir is provided
```

Comment:
Let's add a comment on why we are using native building on Apple Silicon (instead of the Docker approach on Intel)